__tests__/postArticleComment.test.js

```diff
@@ -0,0 +1,56 @@
+const request = require("supertest");
+const app = require("../index");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+const jestSorted = require("jest-sorted");
+
+afterAll(() => {
+    db.end();
+  });
+  
+  beforeEach(() => {
+    return seed(data);
+  });
+
+  describe('POST /api/articles/:article_id/comments', () => {
+    it('Checks if the user has been created', async () => {
+
+      const commentBody = 'BIG DOG';
+      const username = 'lurker';
+      const articleId = 3;
+
+      const response = await request(app)
+        .post(`/api/articles/${articleId}/comments`)
+        .send({ username, body: commentBody });
+
+      expect(response.status).toEqual(201);
+      expect(response.body).toBeInstanceOf(Object);
+      expect(response.body).toEqual({ comment: commentBody });
+
+      const { rows } = await db.query(`SELECT * FROM comments WHERE author = '${username}' AND body = '${commentBody}';`);
```

Comment:
We shouldn't be using a db query in the test file

Reply:
Took me ages to work out how to do this haha, i wanted to check if the comment has been inserted into the DB and check for it but i suppose we are doing that by checking the res.body aren't we.